### PR TITLE
Make level 2 in autopark project.

### PR DESCRIPTION
**Pull request by aces47**
Add Vehicle class.
Add VehicleHelper class.

### CODE REVIEW

File: AutoParkConsole/Classes/Vehicle.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	{
+
+		private readonly VehicleType _vehicleType;
+		private readonly string _modelName;
+		private string _gosNumber;
+		private int _weight;
+		private readonly int _manufactureYear;
+		private int _mileage;
+		private Color _color;
+		private int _tankCapacity;
```
Comment (UNDogMan) [inline AutoParkConsole/Classes/Vehicle.cs:15]: Maybe, Properties? 23 strings to 8, hm..

File: AutoParkConsole/Classes/VehicleHelper.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	{
+		public static void PrintVehicleArray(Vehicle[] vehicles)
+		{
+			foreach (Vehicle vehicle in vehicles)
+			{
+				Console.WriteLine($"{vehicle}");
+			}
+		}
+
+		public static (Vehicle min, Vehicle max) GetFindMaxAndMinMileageVehicles(Vehicle[] vehicles)
```
Comment (PushMeNow) [inline AutoParkConsole/Classes/VehicleHelper.cs:19]: GetFind.... Choose one variant pls in the name of method.

File: AutoParkConsole/Classes/Vehicle.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+			_manufactureYear = manufactureYear;
+			_mileage = mileage;
+			_color = color;
+			_tankCapacity = tankCapacity;
+		}
+
+		public string GetVehicleType() => $"{_vehicleType}";
+		public string GetModelName() => $"{_modelName}";
+		public void SetGosNumber(string gosNumber) => _gosNumber = gosNumber;
+		public string GetGosNumber() => $"{_gosNumber}";
```
Comment (UNDogMan) [inline AutoParkConsole/Classes/Vehicle.cs:45]: Gutters return same strings to your fields.

**Review by UNDogMan — CHANGES_REQUESTED** [review]
Check my comments.

File: AutoParkConsole/Classes/Vehicle.cs [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+		public void SetWeight(int weight) => _weight = weight;
+		public int GetWeight() => _weight;
+		public int GetManufactureYear() => _manufactureYear;
+		public void SetMileage(int mileage) => _mileage = mileage;
+		public int GetMileage() => _mileage;
+		public void SetColor(Color color) => _color = color;
+		public Color GetColor() => _color;
+		public void SetTankCapacity(int tankCapacity) => _tankCapacity = tankCapacity;
+		public int GetTankCapacity() => _tankCapacity;
```
Comment (PushMeNow) [inline AutoParkConsole/Classes/Vehicle.cs:54]: Lets rebuild all these method in props. Don't look at tutorial requirement.

File: AutoParkConsole/Classes/VehicleHelper.cs [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+namespace AutoParkConsole.Classes
+{
+	internal static class VehicleHelper
+	{
+		public static void PrintVehicleArray(Vehicle[] vehicles)
+		{
+			foreach (Vehicle vehicle in vehicles)
+			{
+				Console.WriteLine($"{vehicle}");
```
Comment (PushMeNow) [inline AutoParkConsole/Classes/VehicleHelper.cs:15]: Why are you using string interapolation everywhere?? It is redundant. Use ToString.

**Review by PushMeNow — CHANGES_REQUESTED** [review]

File: AutoParkConsole/Classes/Vehicle.cs (hunk 1 above)
Comment (aces47) [reply AutoParkConsole/Classes/Vehicle.cs]: Fixed.

File: AutoParkConsole/Classes/VehicleHelper.cs (hunk 2 above)
Comment (aces47) [reply AutoParkConsole/Classes/VehicleHelper.cs]: Fixed

File: AutoParkConsole/Classes/Vehicle.cs (hunk 4 above)
Comment (aces47) [reply AutoParkConsole/Classes/Vehicle.cs]: Fixed

File: AutoParkConsole/Classes/VehicleHelper.cs (hunk 5 above)
Comment (aces47) [reply AutoParkConsole/Classes/VehicleHelper.cs]: Fixed

**Review by PushMeNow — APPROVED** [review]

**Review by UNDogMan — APPROVED** [review]

**Pull request merged**